lib/ansible/modules/cloud/ovirt/ovirt_permissions.py

```diff
@@ -158,7 +162,11 @@ def _object_service(connection, module):
             )
         object_id = sdk_object.id
 
-    return objects_service.service(object_id)
+    object_service = objects_service.service(object_id)
+    if module.params['quota_name']:
```

Comment:
Maybe worth to add some check if quota_name is passed and `object_type` != `data_center` fail with error, that we can't assign quota without specifying datacenter.